blog/2024-01-24-embed-figma/index.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Embed Figma in Stately
+description: You can now embed Figma frames in Stately states
+tags: [stately, studio, figma, design, statecharts, state machines, xstate]
+authors: [nick]
+date: 2024-01-24
+slug: 2024-01-24-embed-figma
+image: /blog/2024-01-24-embed-figma/figma-dark.png
+---
+
+We already kicked off 2024 with several major updates to Stately such as inspecting live XState code, syncing with GitHub, and writing code for actions and actors in the Studio. Now we’re excited to announce that we've also launched an integration with Figma.
+
+You can now attach frames in Figma to states in Stately.
```

Comment:
```suggestion
You can now attach Figma frames to states in Stately.
```

I love the symmetry of the original sentence, but I think this might be a bit easier to understand?